db/fixtures/ae_datastore/ManageIQ/Deployment/ContainerProvider/System/StateMachine/Deployment.class/__methods__/create_ansible_inventory.rb

```diff
@@ -1,41 +1,31 @@
-require 'net/ssh'
-
 INVENTORY_FILE = 'inventory.yaml'.freeze
 RHEL_SUBSCRIBE_INVENTORY = 'rhel_subscribe_inventory.yaml'.freeze
 
 def create_ansible_inventory_file(subscribe = false)
   if subscribe
-    template = $evm.root['automation_task'].automation_request.options[:attrs][:rhel_subscribe_inventory]
+    template = $evm.root['rhel_subscribe_inventory']
     inv_file_path = RHEL_SUBSCRIBE_INVENTORY
   else
     $evm.log(:info, "********************** #{$evm.root['ae_state']} ***************************")
-    template = $evm.root['masters'] = $evm.root['automation_task'].automation_request.options[:attrs][:inventory]
+    template = $evm.root['inventory']
     inv_file_path = INVENTORY_FILE
   end
-  begin
-    $evm.log(:info, "creating #{inv_file_path}")
-    File.open(inv_file_path, 'w') do |f|
-      f.write(template)
-    end
-    $evm.root['ae_result'] = "ok"
-    $evm.root['automation_task'].message = "successfully created #{inv_file_path}"
-  rescue StandardError => e
-    $evm.root['ae_result'] = "error"
-    $evm.root['automation_task'].message = "failed to create #{inv_file_path}: " + e
-  end
+  $evm.log(:info, "creating #{inv_file_path}")
+  File.write(inv_file_path, template)
```

Comment:
You'll run into issues with this.  The next state in the state machine is not guaranteed to run on the same appliance.